requirements-ci.txt

```
@@ -1,4 +1,4 @@
 numpy
-pennylane
+git+https://github.com/PennyLaneAI/pennylane.git
```

the changes here don't need the latest to work
```suggestion
pennylane
```